attachments/utils.py

```diff
@@ -14,6 +14,12 @@
 import json
 import uuid
 
+def get_template_path():
+    try:
+        apps.get_app_config('bootstrap')
+    except:
+        return 'attachments/'
+    return 'attachments/bootstrap/'
```

Comment:
Instead of the try/except, can't we just called is_installed?

```python
def get_template_path():
    return 'attachments/bootstrap/' if app.is_installed('bootstrap') else 'attachments/'
```

Reply:
Side question, could this function take an optional filename that will return the joined path? Might make it easier than constantly doing string concat everywhere this is called.

```python
def get_template_path(filename=''):
     prefix = 'attachments/bootstrap/' if app.is_installed('bootstrap') else 'attachments/'
     return f'{prefix}{filename}'

# usage
get_template_path("list.html")
```

Reply:
I made the change you suggested. Although In most places I call get_template_path() once at the top of the file so I have to do the concatenation anyway.